app/controllers/test/IncorpIdApiStubController.scala

```diff
@@ -99,7 +99,7 @@ class IncorpIdApiStubController @Inject()(mcc: MessagesControllerComponents,
           companyNumber = data.crn,
           ctutr = if (!journeyId.contains(charitableOrg)) data.utr else None,
           chrn = if (journeyId.contains(charitableOrg)) data.chrn else None,
-          dateOfIncorporation = Some(LocalDate.of(2020, 1, 1)),
+          dateOfIncorporation = Some(LocalDate.now),
```

Comment:
could these dates cause problems in the ATs or in the envs when trying to test things?

Reply:
that's why i changed the dates in the ATs and stubs